en/firefox_privacy_notice.md

```diff
@@ -85,28 +86,30 @@ Read the full documentation [here](https://firefox-source-docs.mozilla.org/toolk
 
 ## If you use these features, Firefox will share data to provide you functionality and help us improve our products and services: {: #optional-features }
 
-### Search
+### Firefox Suggest 
+
+Mozilla is developing a new feature that helps deliver you better information with less effort. It’s called Firefox Suggest and an early version of it is currently available to users in the United States. Firefox Suggest shows you recommended and sponsored content. We take measures to limit our and our partners’ ability to identify you. [Learn more](https://support.mozilla.org/en-US/kb/navigate-web-faster-firefox-suggest?as=u&utm_source=inproduc), including how to enable and disable this feature.
+
+To improve your browsing experience and to improve the product, Firefox shares the following information:
+
+* __Searches__: Firefox sends Mozilla what you type into the search bar and Mozilla may share that data with its [partners](https://support.mozilla.org/en-US/kb/navigate-web-faster-firefox-suggest?as=u&utm_source=inproduc#w_who-are-mozillas-partners). {: #searches }
```

Comment:
I've tried this on my local copy of bedrock, and this isn't working. I think that these ways of adding the ID to the tag only works for headings. So we may have to add this to `### Firefox Suggest` above. The link to this anchor above does do the right thing, but doesn't work at the moment because this isn't adding the ID.